README.md

```diff
@@ -16,16 +16,50 @@ The major areas currently covered are:
 
 # How to use it?
 
-Distributions can include a systemd timer which will executed the test nightly and log the output to /var/log/ipa/healthcheck. This can be the input into a monitoring system to track changes over time or to alert if a test goes from working to error or warning.
+The simplest way to use Healthcheck is to run it from the command-line as root as ipa-healthcheck. Running from the command-line will display the output to the console unless --output-file=FILENAME is used.
+There is output for _all_ tests so we can be sure that an error condition isn't providing a false positive. The command-line option --failures-only will skip printing the SUCCESS conditions.
 
-It can be run from the command-line as root as ipa-healthcheck. Running from the command-line by default will display the output to the console.
+To automate running Healthcheck every day a systemd timer can be used. 
+The default destination directory for healthcheck logs is `/var/log/ipa/healthcheck` and this can be the input into a monitoring system to track changes over time or to alert if a test goes from working to error or warning.
 
-There is output for _all_ tests so we can be sure that an error condition isn't providing a false positive. The command-line option --failures-only will skip printing the SUCCESS conditions.
+A systemd timer is shipped by RHEL and Fedora packages but not enabled. To enable it:
```

Comment:
The timer can be shipped on other distros as well. I'd change it to something like "A systemd timer is provided but is not enabled by default. To enable it:"

Reply:
Fixed, thanks!